depends/common/imagemagick/0001-prevent-C-system-calls-for-Kodi.patch

```diff
@@ -0,0 +1,34 @@
+From 01ccdb167b15f67a4f5b793d90b9e6189353b15c Mon Sep 17 00:00:00 2001
+From: Alwin Esch <alwin.esch@web.de>
+Date: Mon, 11 Mar 2019 16:01:21 +0100
+Subject: [PATCH] prevent 'C' system calls for Kodi
+
+---
+ imagemagick/magick/delegate.c | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git a/imagemagick/magick/delegate.c b/imagemagick/magick/delegate.c
+index 6d9b810..b95d0fa 100644
+--- a/magick/delegate.c
++++ b/magick/delegate.c
+@@ -376,6 +376,7 @@ MagickExport int ExternalDelegateCommand(const MagickBooleanType asynchronous,
+     (void) ConcatenateMagickString(sanitize_command,"&",MaxTextExtent);
+   if (message != (char *) NULL)
+     *message='\0';
++#if !defined(BUILD_FOR_KODI)
```

Comment:
ImageMagick must not know that a program named Kodi exists. Even in our depends setup.
Just remove the the code in the patch file, as it will never be used for anything else then predepends for screensavers.rsxs.